src/css/search.css

```diff
@@ -0,0 +1,58 @@
+/* Search Bar */
+.ui.fluid.dropdown {
```

Comment:
For whatever reason, the `dropdown.min.css` file gets rid of the border on hover. If you just add `.ui.fluid.dropdown:hover` to this line, our styles will override that and we won't get that behavior.

Reply:
Did this get addressed?

Reply:
Yup! And if I didn't force-merge, it would clear that I did! Almost like that's what the tool was expecting me to do!